src/FindStmtCost.h

```diff
@@ -0,0 +1,147 @@
+#ifndef FINDSTMTCOST_H
+#define FINDSTMTCOST_H
+
+#include "Error.h"
+#include "ExternFuncArgument.h"
+#include "Function.h"
+#include "IRVisitor.h"
+#include "Module.h"
+
+#include <stdexcept>
+#include <unordered_map>
```

Comment:
Be aware that Halide needs to be able to compile with exceptions enabled *and* disabled -- and I don't see what this is used for. Was it included accidentally?